grasa_event_locator/views.py

```diff
@@ -101,47 +111,57 @@ def allUsers(request):
     if request.user.is_authenticated and request.user.userinfo.isAdmin and not request.user.userinfo.isPending:
         userList = userInfo.objects.filter(isPending=False).filter(isAdmin=False)
         context = {'userList': userList}
-
         if request.method == 'POST':
-                #TODO: Change to hostname config value here later...
-
-                send_email(
-                    [request.POST.get('emailAddr')],
-                    render_to_string("messaging/invite_provider_subject.txt"),
-                    render_to_string("messaging/invite_provider_mail.txt"))
+                if request.POST.get('emailAddr'):
+                    try:
+                        # TODO: Change to hostname config value in email later...
+                        send_email(
+                            [request.POST.get('emailAddr')],
+                            render_to_string("messaging/invite_provider_subject.txt"),
+                            render_to_string("messaging/invite_provider_mail.txt"))
+                        context = {'userList': userList, 'sent_invite': True}
+                    except SMTPRecipientsRefused:
+                        context = {'userList': userList, 'invite_failure': True}
```

Comment:
In my own testing, I wasn't able to trigger this exception. Does front-end validation for the user email already cover this? I looked up the [Python docs](https://docs.python.org/3/library/smtplib.html#smtplib.SMTPRecipientsRefused) for this one and I'm confused.

Adding a comment above the exception explaining the real-world criteria to trigger it is helpful.

Reply:
Hey-ya, @jwflory, this is similar to the last time; the invite admin text input field will take admin@admin, so the exception is in there so that the user doesn't dead end at an error page.